lib/tasks/themes.rake

```diff
@@ -369,6 +362,14 @@ task "themes:deduplicate_horizon" => :environment do |task, args|
             &.dig(:user_selectable),
       )
     end
+    if remote_horizon_theme.color_scheme.theme_id == remote_horizon_theme.id
```

Comment:
Since we are calling `remote_horizon_theme.color_scheme` which means the foreign key being used here is `theme_id`. Therefore, I think `remote_horizon_theme.color_scheme.theme_id == remote_horizon_theme.id` will always evaluate to true?

Reply:
No. In this `if` I want to ensure that the default remote horizon theme colour scheme is the theme color scheme.

The theme color scheme can not be edited, and I can map it to the system theme color schemes. 

We might have a situation where the scheme is duplicated and modified. In that case, I want to reassign that new scheme to system theme (else condition).

We have:
Theme#color_scheme_id (default color scheme for theme)
but also ColorScheme#theme_id (many color schemes "belong" to a specific theme)

Does my explanation make sense?